sdk/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -33,8 +33,9 @@ AZ_NODISCARD AZ_INLINE az_span az_keyvault_client_constant_for_application_json(
 az_keyvault_keys_client_options const AZ_KEYVAULT_CLIENT_DEFAULT_OPTIONS
     = { .service_version = AZ_CONST_STR("7.0"),
```

Comment:
Rick was supposed to remove .service_version. Can you follow up with him?

Reply:
Will do.

Reply:
@gilbertw , can I remove the service version?

Reply:
I fixed the service version for storage.  I'll submit a fix for this in KeyVault

Reply:
@antkmsft , https://github.com/Azure/azure-sdk-for-c/pull/322

Reply:
I've submitted a change to master which addresses this.